packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/user-pool-group-stack-transform.ts

```diff
@@ -192,7 +189,9 @@ export class AmplifyUserPoolGroupTransform extends AmplifyCategoryTransform {
         sandbox: {},
       });
       try {
-        sandboxNode.run(overrideCode).override(this._userPoolGroupTemplateObj as AmplifyUserPoolGroupStack & AmplifyStackTemplate);
+        await sandboxNode
+          .run(overrideCode)
+          .override(this._userPoolGroupTemplateObj as AmplifyUserPoolGroupStack & AmplifyStackTemplate);
```

Comment:
This is executing the compiled code in a sandbox. It calls the 'override' function exported by the file/module.